Makefile

```diff
@@ -147,6 +158,10 @@ run-all: ## run the whole stack
 	$(COMPOSE_UP) api keycloak metabase notebook opendata dashboard
 .PHONY: run-all
 
+run-api: ## run the api service
+	$(COMPOSE_UP) api
+.PHONY: run-api
```

Comment:
You should better rename the `run` rule to `run-api` instead.

Reply:
Are you talking about this `run` rule?

```
run: ## run the api server (and dependencies)
	$(COMPOSE_UP) --wait api
.PHONY: run
```

Reply:
Yup.